evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -13,7 +13,7 @@
                         <tr>
                             <th style="width: 80%">{% trans 'Text answer' %}</th>
                             <th style="width: 5%"></th>
-                            <th style="width: 15%">{% trans 'Publish' %}</th>
+                            <th style="width: 15%">{% trans 'Verdict' %}</th>
```

Comment:
Maybe "Action" or "Option"? Or keeping "Publish"/"Publishing"?

Reply:
@He3lixxx and I both think that "verdict" is really funny :D I think that "publish" is a bit weird since it is the same as one of the options. Something like "When evaluation is published..." is a bit long :thinking:

Reply:
I would go for "Action" or "Decision"

Reply:
Decision it is :)